app/src/main/java/org/gdg/frisbee/android/eventseries/TaggedEventSeries.java

```diff
@@ -14,25 +19,53 @@
     private int mDrawerIconResId;
     private int mTitleResId;
     private int mDescriptionResId;
+    private int mDefaultIconResId;
     private int mLogoResId;
-    private int mColorPrimary;
+    private int mSpecialEventTheme;
     private DateTime mStartDateInMillis;
     private DateTime mEndDateInMillis;
 
-    public TaggedEventSeries(String tag, 
+    public TaggedEventSeries(Context context,
+                             @StyleRes int specialEventTheme,
+                             @NonNull String tag,
+                             DateTime startDateInMillis,
+                             DateTime endDateInMillis) {
+
+        final ContextThemeWrapper themeWrapper = new ContextThemeWrapper(context, specialEventTheme);
```

Comment:
This only works for programmed themes, correct? If we get the data from the server we don't need this. Hence it might make more sense to wrap the ids in a java object and create this either from the programmed theme or from server data.